scripts/generate_code.py

```diff
@@ -179,6 +179,38 @@ def glob(path, pattern):
     data="monsterdata_test.json",
 )
 
+flatc(
+    NO_INCL_OPTS
+    + CPP_OPTS
+    + CS_OPTS
+    + [
+        "--binary",
```

Comment:
Why all these other languages being included? It makes the PR very large to review

Reply:
I think it might be a good idea to include other generators in this including type tests. It might help increase test coverage. for instance, I have seen some minor bugs in go generator by running this test. ( #7629 and #7626 ).
Maybe I should add more generator to this test :)

Reply:
I'm fine with adding them, but I was just suggesting to do it in a follow up PR, as it makes this PR harder to review.